src/resources/MachineLearning/ModelConfiguration/ModelConfiguration.ts

```diff
@@ -1,31 +1,23 @@
 import API from '../../../APICore';
-import {LanguageCode, ModelConfigFileType} from '../../Enums';
 import Resource from '../../Resource';
 import {
     AdvancedRegistrationConfigFile,
     AdvancedRegistrationConfigFileCreationResponse,
-    ModelConfigurationUpdateArgs,
 } from './ModelConfigurationInterfaces';
 
 export default class ModelConfiguration extends Resource {
-    static getBaseUrl = (modelId: string, modelConfigFileType: ModelConfigFileType) =>
-        `/rest/organizations/${API.orgPlaceholder}/machinelearning/models/${modelId}/configs/${modelConfigFileType}`;
+    static getBaseUrl = (modelId: string) =>
+        `/rest/organizations/${API.orgPlaceholder}/machinelearning/models/${modelId}/configs`;
 
-    get(modelId: string, modelConfigFileType: ModelConfigFileType, languageCode?: LanguageCode) {
-        return this.api.get<AdvancedRegistrationConfigFile>(
-            this.buildPath(ModelConfiguration.getBaseUrl(modelId, modelConfigFileType), {languageCode})
-        );
+    getAdvancedConfig(modelId: string) {
+        return this.api.get<AdvancedRegistrationConfigFile>(`${ModelConfiguration.getBaseUrl(modelId)}/advanced`);
     }
 
-    update(
-        modelId: string,
-        modelConfigFileType: ModelConfigFileType,
-        {modelConfigFileContents, languageCode}: ModelConfigurationUpdateArgs
-    ) {
+    updateAdvancedConfig(modelId: string, modelConfigFileContents: string) {
         return this.api.put<AdvancedRegistrationConfigFileCreationResponse>(
-            this.buildPath(ModelConfiguration.getBaseUrl(modelId, modelConfigFileType), {languageCode}),
+            `${ModelConfiguration.getBaseUrl(modelId)}/advanced`,
             modelConfigFileContents,
-            {method: 'put', body: modelConfigFileContents, headers: {'Content-Type': 'text/plain'}}
+            {method: 'put', body: modelConfigFileContents, headers: {'Content-Type': 'application/json'}}

```

Comment:
These are the default values IIRC
```suggestion
```

Reply:
The default args are `args: RequestInit = {method: 'put', body: JSON.stringify(body), headers: {'Content-Type': 'application/json'}}` https://github.com/coveo/platform-client/blob/master/src/APICore.ts#L63
In this specific case, we don't need `JSON.stringify(body)`, otherwise we'll get something like `""{}""` with repetitive double quotes.